### ISSUE TITLE
Cloud / local wallet login / wallet /account model howto / documents / tutorial

### ISSUE THREAD

**Issue by abitmore** [bitshares/bitshares-ui#758]
Lots of confusions here about why unable to login: https://bitsharestalk.org/index.php/board,45.0.html . I guess most of them are due to our ever changing UI/UX. If we have some good documents or tutorials, please post in the forum and stick it.

**Comment by btsfav** [comment]
see https://github.com/bitshares/bitshares-ui/issues/577

**Comment by btsfav** [comment]
+ an AI bot for support requests would be super cool

**Comment by wmbutler** [comment]
Might make sense for someone to extend this area inside the wallet:

https://wallet.bitshares.org/#/help/introduction/wallets

**Comment by pnomolos** [comment]
@wmbutler I'll take this since I was recently banging my head against the wall while dealing with it anyway.  Are you looking for more integrated help documentation?  A standalone tutorial/blog post?  A bit of both? :)

**Comment by wmbutler** [comment]
Probably something in the help area that walks through the steps to take when troubleshooting cloud vs local wallets. FarmerD has a lot of experience walking ppl through this. For instance the Cloud wallet has 45 characters and starts with a "P". These types of hints can help people quickly determine which type of wallet they have. Take time to conduct interviews and get best practices.

It will also be important to make this help document easily accessible from the problem areas.

**Comment by pnomolos** [comment]
@wmbutler I'm looking to help out the documentation team, primarily in the UI area - is this something that would be better just waiting for that as opposed to doing it here?

**Comment by wmbutler** [comment]
We have a help area, so it might not hurt to link to it with a new tab in the wallet, Maybe you can mirror the procedure from the documentation team so we are all communicating the same message.

**Comment by wmbutler** [comment]
@pnomolos any chance you'd like to take the charge of linking circle-i references to the the official bts documentation and helping us get rid of the in wallet help?

**Comment by pnomolos** [comment]
I’m currently away on a trip, but when I’m back next week I can take a look
at that :)

**Comment by pnomolos** [comment]
@wmbutler Just confirming that I’m still on deck to get this done :)

**Comment by wmbutler** [comment]
Yes, please. Maybe do a couple of circle I references linked to the appropriate documentation area that I suggested and we can comment on it before we move forward with the entire site?

**Comment by clockworkgr** [comment]
@pnomolos any progress?

**Comment by pnomolos** [comment]
@clockworkgr Not really, work and real life has become far too busy.  I always hold out hope I'll get more time to get back to this but if you want to take it feel free to :)
